main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       bartender
+//  action 1:   the bartender takes an order
+bartender.takeOrder();
+//  action 2:   the bartender shakes a drink
+bartender.shakeDrink();
+//  action 3:   the bartender serves a drink
+bartender.serveDrink();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       politician
+//  action 1:   the politician campaigns
+politician.campaign();
+//  action 2:   the politician gives a speech
+politician.giveSpeech();
+//  action 3:   the politician argues
+politician.argue();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       chameleon
+//  action 1:   the chameleon hides
+chameleon.hide();
+//  action 2:   the chameleon eats a plant
+chameleon.eatPlant();
+//  action 3:   the chameleon sleeps
+chameleon.sleep();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       AI
+//  action 1:   the AI learns
+ai.learn();
+//  action 2:   the AI questions
+ai.question();
+//  action 3:   the AI makes a joke
+ai.makeJoke();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       oven
+//  action 1:   the oven heats up
+oven.heatUp();
+//  action 2:   the oven cools down
+oven.coolDown();
+//  action 3:   the oven burns food
+oven.burnFood();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       wormhole
+//  action 1:   the wormhole opens
+wormhole.open();
+//  action 2:   the wormhole defies physics
+wormhole.defyPhysics();
+//  action 3:   the wormhole closes
+wormhole.close();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       janitor
+//  action 1:   the janitor sweeps the floor
+janitor.sweepFloor();
+//  action 2:   the janitor collects garbage
+janitor.collectGarbage();
+//  action 3:   the janitor takes a break
+janitor.takeBreak();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message